Makefile

```diff
@@ -219,14 +219,14 @@ deploy: clean-helm-rendered
 slim-deploy: clean-helm-rendered
 	@echo "+ $@"
 	kubectl create namespace stackrox || true
-	helm template scanner chart/ --set scannerImage=us.gcr.io/stackrox-ci/scanner-slim,scannerDBImage=us.gcr.io/stackrox-ci/scanner-db-slim,tag=$(TAG),logLevel=$(LOGLEVEL),updateInterval=2m --output-dir rendered-chart
+	helm template scanner chart/ --set scannerImage=quay.io/stackrox-io/scanner-slim,scannerDBImage=quay.io/stackrox-io/scanner-db-slim,tag=$(TAG),logLevel=$(LOGLEVEL),updateInterval=2m --output-dir rendered-chart
 	kubectl apply -R -f rendered-chart
 
 .PHONY: deploy-dockerhub
 deploy-dockerhub: clean-helm-rendered
```

Comment:
just noticed this. Do you recall what this is for @connorgorman ?

Reply:
oh deploying locally

Reply:
👍 seems fine to keep then

Reply:
maybe revert the change here, though since `make image` will build `stackrox/scanner`

Reply:
yeah good point

Reply:
well on a quick second thought, you wouldn't be able to deploy CI cause there is no dockerhub. Should just tag with OS then

Reply:
CI uses `make deploy` or `make deploy-slim` which you tag properly

Reply:
maybe we rename this to `deploy-local`?